code/main/task/eventProcessor.brs

```diff
@@ -194,6 +195,16 @@ function _adb_EventProcessor(task as object) as object
             end if
         end function,
 
+        _getECID: function(event as object) as void
+            ecid = m._identityModule.getECID()
+
+            _adb_logDebug("EventProcessor::_getECID() - Dispatching getECID response event with ECID: (" + ecid + ")")
```

Comment:
The retrieved `ecid` could be invalid, should we use `FormatJson(ecid)` instead?